### PR TITLE
feat: Add docs for Shopify Hydrogen

### PR DESCRIPTION
**Pull request by AbhiPrasad**
## DESCRIBE YOUR PR
*Tell us what you're changing and why. If your PR **resolves an issue**, please link it so it closes automatically.*

## IS YOUR CHANGE URGENT?  

Help us prioritize incoming PRs by letting us know when the change needs to go live.
- [ ] Urgent deadline (GA date, etc.): 
- [ ] Other deadline: 
- [x] None: Not urgent, can wait up to 1 week+

## SLA

- Teamwork makes the dream work, so please add a reviewer to your PRs.
- Please give the docs team up to 1 week to review your PR unless you've added an urgent due date to it.
Thanks in advance for your help!

## PRE-MERGE CHECKLIST

*Make sure you've checked the following before merging your changes:*

- [ ] Checked Vercel preview for correctness, including links
- [ ] PR was reviewed and approved by any necessary SMEs (subject matter experts)
- [ ] PR was reviewed and approved by a member of the [Sentry docs team](https://github.com/orgs/getsentry/teams/docs)

### CODE REVIEW

**Review by lforst — APPROVED** [review]
A bit of a wild one but probably very useful

File: docs/platforms/javascript/guides/remix/frameworks/hydrogen.mdx [hunk 1]
```diff
@@ -0,0 +1,74 @@
...
+
+If the setup through the wizard doesn't work for you, you can also [set up the Remix SDK manually](/platforms/javascript/guides/remix/manual-setup/).
+
+After installing the Sentry Remix SDK, delete the newly generated `instrumentation.server.mjs` file and all newly generated code from `entry.server.tsx`. This instrumentation is not needed because you are going to use the Sentry Cloudflare SDK for server-side instrumentation.
+
+Now you can install the Sentry Cloudflare SDK. First, install the SDK with your package manager:
+
+<PlatformContent includePath="getting-started-install" />
+
+Then update your `server.ts` file to use the `wrapRequestHandler` method
```
Comment (coolguyzone) [inline docs/platforms/javascript/guides/remix/frameworks/hydrogen.mdx:22]:
```suggestion
Then update your `server.ts` file to use the `wrapRequestHandler` method:
```

File: docs/platforms/javascript/guides/remix/frameworks/hydrogen.mdx [hunk 2]
```diff
@@ -0,0 +1,74 @@
...
+      },
+      async () => {
+        // request handling logic
+      }
+    );
+  },
+};
+```
+
+To remove errors relating to `node:async_hooks` (which is included in the sdk, but not used by `wrapRequestHandler`), add a custom vite plugin to your `vite.config.ts` file that will alias it to an empty module.
```
Comment (coolguyzone) [inline docs/platforms/javascript/guides/remix/frameworks/hydrogen.mdx:54]:
```suggestion
To remove errors relating to `node:async_hooks` (which is included in the sdk, but not used by `wrapRequestHandler`), add a custom vite plugin to your `vite.config.ts` file that will alias it to an empty module:
```

File: docs/platforms/javascript/guides/remix/frameworks/hydrogen.mdx [hunk 3]
```diff
@@ -0,0 +1,74 @@
...
+
+If you're using the Shopify Hydrogen framework, you can use the Sentry Remix SDK to add Sentry instrumentation to your app.
+
+First, install the Sentry Remix SDK in your application. We recommend using the Sentry wizard to automatically install the SDK:
+
+```bash
+npx @sentry/wizard@latest -i remix
+```
+
+If the setup through the wizard doesn't work for you, you can also [set up the Remix SDK manually](/platforms/javascript/guides/remix/manual-setup/).
```
Comment (coolguyzone) [inline docs/platforms/javascript/guides/remix/frameworks/hydrogen.mdx:14]:
```suggestion
If the wizard doesn't work for you, you can also [set up the Remix SDK manually](/platforms/javascript/guides/remix/manual-setup/).
```

**Review by coolguyzone — APPROVED** [review]
Thanks for adding! Left a couple of suggestions but looks good to me.

**Pull request merged**